set.mm

```diff
@@ -20804,10 +20805,11 @@ theorem as an axiom of set theory (Axiom 0 of [Kunen] p. 10).  In the
     spime.2 $e |- ( x = y -> ( ph -> ps ) ) $.
     $( Existential introduction, using implicit substitution.  Compare Lemma 14
        of [Tarski] p. 70.  Usage of this theorem is discouraged because it
-       depends on ~ ax-13 .  Check out ~ spimew for a weaker version requiring
-       fewer axioms.  (Contributed by NM, 7-Aug-1994.)  (Revised by Mario
-       Carneiro, 3-Oct-2016.)  (Proof shortened by Wolf Lammen, 6-Mar-2018.)
-       (New usage is discouraged.) $)
+       depends on ~ ax-13 .  See ~ spimew and ~ spimevw for weaker versions
+       requiring fewer axioms.  (Contributed by NM, 7-Aug-1994.)  (Revised by
+       Mario Carneiro, 3-Oct-2016.)  (Proof shortened by Wolf Lammen,
+       6-Mar-2018.)  Usage of this theorem is discouraged because it depends on
```

Comment:
`Usage of this theorem is discouraged because it depends on ~ ax-13` is written two times.